sphinx-mode.el

```diff
@@ -123,49 +122,54 @@ If BUFFER is not given use the `current-buffer'."
     (find-file (plist-get target :file))
     (goto-char (plist-get target :point))))
 
-(defun sphinx-compile-find-makefile-dir () ; adapted from rst-compile-find-conf
-  "Look for the Sphinx Makefile in parents of the current path and returns the directory which contains it as a string."
+(defun sphinx-compile ()
+  "Runs 'make' in project root directory, prompting user for target format. Run 'make help' to see list of target formats."
```

Comment:
Use `Run` instead of `Runs` (imperative over indicative), that's the style Emacs follows.

Also try to keep the first line under 80 chars and the rest in a separate paragraph.  I think the second line can be on a new line (separated by space).

You can run `checkdoc` command in Emacs and it will provide some suggestions.